jinja/carnival/talent_show.html

```diff
@@ -12,53 +12,64 @@
         </div>
         <div class="row text-center">
             <h3 class="col-12 mt-5 mb-5 grey-text" style="font-family: 'Roboto', sans-serif !important;">
-            Show the world what you can do in our Carnival <strong>Talent Show</strong>.
+            Congratulations to the 🏆 winners 🏆 of our Quantum Carnival <strong>Talent Show</strong>.
+            <br>
+            <p>Check out the best submissions below.    </p>
             </h3>
-	<div style="font-size: 14pt;">
-            <p>
-            Have you ever wanted to participate in the PennyLane community without writing code? Do you love our hand-drawn images and want to explore your artistic side too 🖌️?
-This is your chance! 
-</p>
+        </div>
+            
+            <div class = "row justify-content-center align-items-center">    
+                <div class="col">            
+                    <img src="../img/draw__pennylane.png" class="rounded mx-auto d-block" style="max-width: 300px;"></img>
+                    <figcaption style="text-align: center;">Alberto Maldonado Romo</figcaption>
```

Comment:
Is it OK to have a person win twice?

Reply:
Yes, it's ok. He submitted two very good pieces.